### DISCUSSION TITLE
Shuffling Artists

### DISCUSSION THREAD

**Discussion by AlperShal** [navidrome/navidrome#2448] [Ideas]
I understand that shuffling all of the library would cause some serious problems but not having shuffle or play all option in an artist page is a really serious lacking feature I think. Every time I open Navidrome I am going to an artist page and every time it's lacking the option to play/shuffle songs of the artist. I can't think of any reason that this can be a problem. If so, please enlighten me.

**Comment by certuna** [comment]
There's no shuffle button yet, but you can go to Songs, type in the artist name, and then shuffle that.

**Comment by AlperShal** [comment]
I have switched to another service because lack of a few features but still thanks for the answer.

**Comment by tyrossel** [comment]
+1, I would also use this feature very often. Have exactly the same action list for artists as for albums (play, play next, shuffle, add to playlist, etc.) would be amazing.
